server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -684,18 +684,17 @@ protected void runInContext() {
                 return;
             }
             // collect the vm disk statistics(total) from hypervisor. added by weizhou, 2013.03.
-            s_logger.trace("Running VM disk stats ...");
-            try {
-                Transaction.execute(new TransactionCallbackNoReturn() {
-                    @Override
-                    public void doInTransactionWithoutResult(TransactionStatus status) {
-                        s_logger.debug("VmDiskStatsTask is running...");
+            s_logger.debug("VmDiskStatsTask is running...");
```

Comment:
why, this change in message and level?

Reply:
@DaanHoogland the message and level are not changed.  moved out from try block.

Removed the line which is not needed as we have debug message.
`s_logger.trace("Running VM disk stats ...");`